lib/dry/rails/container.rb

```diff
@@ -41,6 +41,15 @@ class Container < System::Container
       # @!scope class
       setting :auto_register_paths, [].freeze, reader: true
 
+      # @overload config.import_constant=(import_constant)
+      #   Set a custom import constant name
+      #
+      #   @param import_constant [String]
+      #
+      #   @api public
+      # @!scope class
+      setting :import_constant, 'Import'.freeze, reader: true
```

Comment:
@diegotoral let's call it `auto_inject_constant` and set its default value to `Deps` (that's how we called it in the upcoming Hanami 2.0)